manim/cli/project/templates/Graph.py

```diff
@@ -0,0 +1,16 @@
+class GraphTemplate(GraphScene):
```

Comment:
This is being deprecated in #1161 which will likely be merged this release cycle. We could use this template, but it will have to be updated; instead, it would make most sense to wait until that PR is merged and use `Axis` instead.

Reply:
yes, it is manim template. Thanks for the review. I will remove this template for now and add another one.

Reply:
After reading #1161 I now understand what Axis means. I will switch the graph scene with Axis.